src/upcoming-components/CheckboxField/CheckboxField.tsx

```diff
@@ -97,8 +97,6 @@ export const CheckboxField = ({
     variant === 'inline' && styles['checkbox-field--inline'],
     size === 'sm' && styles['checkbox-field--sm'],
     inverted && styles['checkbox-field--inverted'],
-    isError && styles['eds-is-error'],
-    disabled && styles['eds-is-disabled'],
```

Comment:
Also wondering here if we need to do something with `disabled` prop because it doesn't seem to be used anywhere else.

Reply:
Yea this case the class styles have no styling attached so I removed these lines. I can go ahead and removed the disabled prop from this component, or just let it be since it may be required if we decide to move such a component to `components` folder

Reply:
Maybe leave a todo comment on the disabled prop that the styling for that prop needs to be added?

Reply:
That makes sense, will do

Reply:
Awesome, thanks!